tempest.scenario.test_network_basic_ops.TestNetworkBasicOps.test_mtu_sized_frames.*: "Timed out waiting for 195.169.23.69 to become reachable"
tempest.scenario.test_network_basic_ops.TestNetworkBasicOps.test_port_security_macspoofing_port.*: "AssertionError: Timed out waiting for 10.100.0.19 to become reachable from 195.169.23.132"
tempest.scenario.test_network_basic_ops.TestNetworkBasicOps.test_subnet_details.*: "Cirros image can't open '/var/run/udhcpc.eth0.pid': No such file or directory"
tempest.api.volume.test_volumes_backup.VolumesBackupsTest.test_volume_backup_create_get_detailed_list_restore_delete.*: "backup 9941879f-0c7d-4fdd-b289-3af6ca02ee51 failed to build and is in ERROR status"
tempest.api.compute.admin.test_aggregates.AggregatesAdminTestJSON.test_aggregate_add_host_create_server_with_az.*: "Fault: {'code': 500, 'created': '2023-03-07T13:39:30Z', 'message': 'No valid host was found. There are not enough hosts available.'}. Server boot request ID: req-a6feda98-ede0-45b8-8048-dedd0bf596a4."
tempest.api.volume.admin.test_volume_types.VolumeTypesTest.test_volume_crud_with_volume_type_and_extra_specs.*: "volume cb204cd3-def3-4b07-abe7-8183fb5657c8 failed to build and is in ERROR status"
tempest.api.volume.test_volume_delete_cascade.VolumesDeleteCascade.test_volume_from_snapshot_cascade_delete.*: "<map object at 0x7f6c18620370> matches Contains('b53af9ad-148f-4580-8a1a-6c23a0113219')"
tempest.scenario.test_server_multinode.TestServerMultinode.test_schedule_to_all_nodes.*: "No valid host was found. There are not enough hosts available."
